dashboard/lib/pd/foorm/workshop_summarizer.rb

```diff
@@ -158,5 +158,16 @@ def self.add_question_to_summary(summary_at_question, answer, answers, question_
       end
       summary_at_question
     end
+
+    # Sort summaries so that survey data is ordered in the order surveys
+    # should be taken in. i.e. Pre-Workshop, Day 1, Day 2,.. Post-Workshop
+    def self.sort_summary(workshop_summary)
+      temp_summary = workshop_summary.sort_by {|survey_key, _| get_index_for_survey_key(survey_key)}
```

Comment:
Do you need the `_` parameter here?

Reply:
If I don't then the single variable will be a 2 element array containing the survey_key and summary, so adding `_` extracts out the survey_key